.github/workflows/build.yml

```diff
@@ -649,3 +652,50 @@ jobs:
           cd test
           pnpm i
           pnpm moonwall test CI_upgrade_RT_${{matrix.chain}}
+
+  zombie_upgrade_test:
+    runs-on: ubuntu-latest
+    needs: ["set-tags", "build"]
+    strategy:
+      matrix:
+        chain: ["moonbase", "moonriver"]
+    env:
+      GH_WORKFLOW_MATRIX_CHAIN: ${{ matrix.chain }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.set-tags.outputs.git_ref }}
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: "pnpm"
+          cache-dependency-path: test/pnpm-lock.yaml
+      - name: "Download runtime"
+        uses: actions/download-artifact@v3.0.2
+        with:
+          name: runtimes
+          path: test/tmp
+      - name: "Download node"
+        uses: actions/download-artifact@v3.0.2
+        with:
+          name: moonbeam
+          path: test/tmp
+      - name: "Run zombie test"
+        run: |
+          cd test
+          pnpm i
+
+          ## Download latest node binaries
+          pnpm moonwall download polkadot latest tmp
+          pnpm moonwall download moonbeam latest tmp/dl/
```

Comment:
Shouldn't we test the local moonbeam ?

Reply:
Purpose of this test is to have a locally built node running an old runtime, upgrading to the locally built runtime. This is what the para tests already currently do, which I'm trying to replicate.

Reply:
ok